lib/internal/test_runner/coverage.js

```diff
@@ -353,6 +354,7 @@ class TestCoverage {
         continue;
       }
       const { data, lineLengths } = sourceMapCache[url];
+      if (!data) throw new ERR_SOURCE_MAP_CANNOT_PARSE(url);
```

Comment:
> The error says that the source map does not exist or cannot be parsed... but there is still an entry for it in the source map cache.

There is an entry, but it's empty. 

---

Before this change, an error was thrown because the entry was empty, and therefore didn't have any properties. Does that error get thrown anywhere else? If so, I can try and handle them as well